### ISSUE TITLE
[Accessibility] Column chart column values are not announced in Android Talkback

### ISSUE THREAD

**Issue by charlie-bud** [highcharts/highcharts#23354]
#### Expected behaviour
When using Android Talkback and focussing on a specific column, we'd expect the value of the column to be announced, for example "Sunday, $78.36. Spending".

#### Actual behaviour
When using Android Talkback and focussing on a specific column, only "Graphic" is announced while other screen readers such as Voiceover and NVDA _do_ announce the values.

#### Live demo with steps to reproduce
Even the Highcharts default template JS Fiddle demonstrates this 👇 

https://jsfiddle.net/highcharts/LLExL/

https://www.loom.com/share/27148364eb3749c6b1b161b188b00804

#### Product version
`v12.3.0`

#### Affected browser(s)
Android using Chrome with Talkback (Screen reader)

**Comment by bre1470** [comment]
Hello @charlie-bud !
Thank you for your report. I forwarded this to the accessibility team.

**Comment by bre1470** [comment]
One question, @charlie-bud: Which Android version was used to test Talkback?

**Comment by charlie-bud** [comment]
Hey @bre1470 - thanks for forwarding this issue on. I personally tested on Android 15.0.

**Comment by charlie-bud** [comment]
Hi @bre1470 - just wanted to follow up and see if there's been any movement on a fix here?

**Comment by bre1470** [comment]
Hello @charlie-bud !
Thank you for reaching out.
We are not able to reproduce the issue with existing Android 15 devices and wait for a new Android device to test it ones more.
Additionally we work on a refactored support of accessibility and expect that these kind of issues will be fixed by it.

**Comment by charlie-bud** [comment]
Hey @bre1470 - I've retested `v12.4.0` using Android 15 & 16 and can confirm this is now resolved. Thank you! 🙇‍♂️

**Comment by bre1470** [comment]
Hello @charlie-bud! Thank you for the feedback! I am glad that the issue has been solved. Happy Coding!
